README.md

```diff
@@ -102,7 +126,14 @@ Using the object literal notation, Define an object called student and give it t
 
 Your Answer:
 ```js
-// write code here
+var student = {
+  firstName: "Andrew",
+  age: 29,
+  sayHello: function() {
+      console.log("Hi, my name is " + student.firstName)
+  }
+}
+console.log(student.sayHello);
```

Comment:
This works, although I would replace `student.firstName` in `sayHello` with `this.firstName`.